drivers/linux/eth/ionic/ionic_ethtool.c

```diff
@@ -775,10 +859,8 @@ static void ionic_get_channels(struct net_device *netdev,
 
 	/* report maximum channels */
 	ch->max_combined = lif->ionic->ntxqs_per_lif;
-	if (!ionic_use_eqs(lif)) {
-		ch->max_rx = lif->ionic->ntxqs_per_lif / 2;
-		ch->max_tx = lif->ionic->ntxqs_per_lif / 2;
-	}
+	ch->max_rx = lif->ionic->ntxqs_per_lif / 2;
+	ch->max_tx = lif->ionic->ntxqs_per_lif / 2;
```

Comment:
Please ignore, this is just me thinking out loud since I had to read this twice. I think it would be nice to have this as:

ch->max_* = ch->max_combined / 2;

Obviously no different functionally, so no big deal.